composer.json

```diff
@@ -10,12 +10,12 @@
     ],
     "require": {
         "guzzlehttp/guzzle": "^6.3",
-        "illuminate/support": "^5.4",
-        "phpunit/phpunit": "~6.0|~7.0|~8.0"
+        "illuminate/support": "^6.0",
```

Comment:
I'm down with not supporting 5.8 anymore since LTS will end for it soon.

Reply:
I don't think it would be possible to support both together either